nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -540,14 +541,19 @@ define([
         }
 
         function stop() {
+            if (!running) {
+                return Promise.resolve();
+            }
```

Comment:
check whether the configure tab is running or not, and only stop the widgets if it is.